Sources/UILocalNotification+Sweet.swift

```diff
@@ -5,7 +5,15 @@
         static let idKey = "idKey"
         static let fireDateKey = "fireDateKey"
 
-        public static func create(_ id: String, fireDate: Date, soundName: String? = nil, message: String, actionTitle: String? = nil) {
+        /// Schedules a local notification.
+        ///
+        /// - Parameters:
+        ///   - id: The id used to represent a local notification in your app.
+        ///   - fireDate: The date when the notification will trigger
+        ///   - soundName: The name of the sound to be used when the notification is received.
+        ///   - message: The message that the app will use for your local notification.
+        ///   - actionTitle: The title of the action to be displayed by your notification, it's optional, though, since Apple provides a good default for this.
+        public static func schedule(using id: String, fireDate: Date, soundName: String? = nil, message: String, actionTitle: String? = nil) {
```

Comment:
Here it feels like it uses "twilight" to register the notification.

```swift
UILocalNotification.schedule(using: "twilight", at: fireDate, message: "Good night")
```

Here it feels like it doesn't convey that. Schedules with a string?

```swift
UILocalNotification.schedule(with: "twilight", at: fireDate, message: "Good night")
```

Maybe it could also be:


```swift
UILocalNotification.schedule(named: "twilight", at: fireDate, message: "Good night")
```